lib/ssl/src/dtls_socket.erl

```diff
@@ -169,15 +175,15 @@ check_active_n(EmulatedOpts, Socket = #sslsocket{pid = {dtls, #config{dtls_handl
             ok
     end.
 
-getopts(_, #sslsocket{pid = {dtls, #config{dtls_handler = {ListenPid, _}}}}, Options) ->
+getopts(_, #sslsocket{socket_handle = {ListenPid, _}, listener_config =#config{}}, Options) ->
     SplitOpts = tls_socket:split_options(Options),
     dtls_packet_demux:get_sock_opts(ListenPid, SplitOpts);
-getopts(gen_udp,  #sslsocket{pid = {Socket, #config{emulated = EmOpts}}}, Options) ->
+getopts(gen_udp,  #sslsocket{socket_handle = {Socket, #config{emulated = EmOpts}}}, Options) ->
```

Comment:
Isn't the second part of the tuple a Port?  Says so above

Reply:
It depends if it is a emulated listen socket or an "ordinary" socket. I can consider making more  structure for these sockets.

Reply:
Actually I think these clauses are dead code, and part of the server-multiplexor code needed upgrading so, I fixed that. Will push new version soon.